tool/plugin/test/plugin_test.dart

```diff
@@ -136,18 +136,18 @@ BuildCommandRunner makeTestRunner() {
 class TestAntBuildCommand extends AntBuildCommand {
   TestAntBuildCommand(runner) : super(runner);
 
-  doit() {}
+  Future<int> doit() async => new Future(() => 0);
```

Comment:
Curious if `new Future.value(0);` is just as good?

Reply:
Ha! I missed that. Looks like another clean-up pass is needed, someday. Too bad there's no lint rule for that.

I sorted the rules and now I can't find that comment to reply to, so this will have to do.